src/modules/Dropdown/Dropdown.js

```diff
@@ -134,6 +134,12 @@ export default class Dropdown extends Component {
     /** Define whether the highlighted item should be selected on blur */
     selectOnBlur: PropTypes.bool,
 
+    /** Make the dropdown options searchable (`boolean` or `function(options, searchQuery) => filteredOptions`) */
```

Comment:
Your proposed description without the signature is great.

I definitely agree a signature is better than no signature.  However, I want to have a consistent mechanism in place for adding them. I plan to extend the props table to handle function signatures via standardized jsdoc tags (i.e. `@param` etc).  I'll add an issue for this now!